time/src/test/java/net/jqwik/time/api/LocalTimeTests.java

```diff
@@ -20,12 +20,28 @@
 		return Times.times();
 	}
 
-	@Property
-	void validLocalTimeIsGenerated(@ForAll("times") LocalTime time) {
-		assertThat(time).isNotNull();
+	@Group
+	class SimpleArbitraries {
+
+		@Property
+		void validLocalTimeIsGenerated(@ForAll("times") LocalTime time) {
+			assertThat(time).isNotNull();
+		}
+
+		@Property
+		void worstCaseTimeGeneration(@ForAll("worstCases") LocalTime time) {
+			assertThat(time).isNotNull();
+		}
+
+		@Provide
+		Arbitrary<LocalTime> worstCases() {
+			return Times.times().between(LocalTime.of(22, 59, 59, 999_999_998), LocalTime.of(23, 00, 00, 000_000_001));
+		}
```

Comment:
Is it a problem when this worst case scenario does not work?

```
timestamp = 2021-01-27T17:24:36.556964900, SimpleArbitraries:worstCaseTimeGeneration = 
  net.jqwik.api.TooManyFilterMissesException:
    Filtering [Filtering [net.jqwik.api.RandomGenerator$$Lambda$403/0x0000000800c6c440@36ebc363]] missed more than 10000 times.
```

Reply:
I think I have an idea: 2 algorithms that generate

Reply:
Between() shouldn’t require filtering IMO so the concrete example failing may not be a problem but the general approach is.

Reply:
The problem is using plusNanos causes a lot of filtering. So I can't use the approach from `DefaultLocalDateArbitrary`.

Reply:
A single generator of just nanos will probably not work. One idea is to have to generators, one for seconds within a day and another one for nanos within a second. Very often the second one will not be used when only second accuracy is required.